drivers/cleanup.py

```diff
@@ -2741,8 +2760,8 @@ def get_state(srUuid):
     locking.
     """
     init(srUuid)
-    if lockRunning.acquireNoblock():
-        lockRunning.release()
+    if lockActive.acquireNoblock():
```

Comment:
This change was not here in the previous version.
What do we want to tell the user: if it is spawned or actively running?